### PR TITLE
test cases for navbar component

### PR DESCRIPTION
**Pull request by iamYashSinha**
### Added Test cases: 
1. renders the Navbar component
2. user whether loggedIn or not
3. renders Navbar Links
4. should navigate to different route when different navbar link is clicked

### CODE REVIEW

File: src/components/navBar/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    test('renders the Navbar component', () => {
+        const { getByTestId } = render(<NavBar />);
+        const navbar = getByTestId('navbar');
+        expect(navbar).toBeInTheDocument();
+    });
+
+    test('user whether loggedIn or not', () => {
+        
+        jest.spyOn(authHooks, 'default').mockImplementation(() => (
+            { userData: { userName: "IamYash",
```
Comment (Maheima) [inline src/components/navBar/index.test.tsx:18]: test file lives under __tests__/unit folder. Please fix that

**Review by Maheima — CHANGES_REQUESTED** [review]

File: src/components/navBar/index.test.tsx (hunk 1 above)
Comment (iamYashSinha) [reply src/components/navBar/index.test.tsx:18]: On it.

**Review by yesyash — APPROVED** [review]

File: __tests__/Unit/Components/Navbar.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { render, fireEvent, screen } from '@testing-library/react'
+import NavBar from '../../../src/components/navBar'
+import * as  authHooks from '@/hooks/useAuthenticated';
+
+
+
```
Comment (sahsisunny) [inline __tests__/Unit/Components/Navbar.test.tsx:7]: remove the extra blank spaces

File: __tests__/Unit/Components/Navbar.test.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                        firstName: "Yash", 
+                         profilePicture: "",
+                        },   
+             isLoggedIn: true,
+             isLoading: true 
+            }
+       ));
+       render(<NavBar />)
+       expect(screen.getByText("Hello, Yash"))
+    })
```
Comment (sahsisunny) [inline __tests__/Unit/Components/Navbar.test.tsx:28]: can we also test navbar links ?

**Review by sahsisunny — COMMENTED** [review]
Description is missing.

**Pull request closed without merging**